rpc/types.go

```diff
@@ -72,11 +72,12 @@ type jsonWriter interface {
 type BlockNumber int64
 
 const (
-	SafeBlockNumber     = BlockNumber(-4)
-	AcceptedBlockNumber = BlockNumber(-3)
-	LatestBlockNumber   = BlockNumber(-2)
-	PendingBlockNumber  = BlockNumber(-1)
-	EarliestBlockNumber = BlockNumber(0)
+	SafeBlockNumber      = BlockNumber(-4)
+	FinalizedBlockNumber = BlockNumber(-3)
+	AcceptedBlockNumber  = BlockNumber(-3)
```

Comment:
do we want these to be same?

Reply:
Should we just remove AcceptedBlockNumber and use Finalized?

Reply:
I also thought about that, but I think we could leave that to a different PR, mostly I added FinalizedBlockNumber back to allow tests to be the same as they are in upstream